examples/demo.yaml

```diff
@@ -102,6 +102,10 @@ spec:
         volumeMounts:
         - mountPath: /mnt/data
           name: data
+      - name: novolumehere
```

Comment:
Can you remove the mount from one of the existing container (for example [the dummy one](https://github.com/DataDog/chaos-controller/blob/a2eb1c5b32178a965b508eaf17c48ea1e8b92360/examples/demo.yaml#L102)) instead of creating a new one? We already have 2 containers in the `demo-curl` pod and none are really using the mount.